stock_picking_portal/models/res_config_settings.py

```
@@ -13,25 +13,37 @@ class ResConfigSettings(models.TransientModel):
+        to_invisible = currently_visible - selected
+        to_visible = selected - currently_visible
+
+        if to_invisible:
+            to_invisible.write({"portal_visible": False})
+        if to_visible:
+            to_visible.write({"portal_visible": True})
+
+        return res
 
     @api.model
     def get_values(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_values`

(ANN201)